sufia-models/app/models/sufia/works/work.rb

```diff
@@ -0,0 +1,7 @@
+# The base class of all works
```

Comment:
I closed this PR since it was against the wrong branch.  That said WorthWhile and Curate both have a concept of many work types.  I left that in.